sympy/printing/jscode.py

```diff
@@ -27,23 +27,41 @@
     'atan': 'Math.atan',
     'atan2': 'Math.atan2',
     'atanh': 'Math.atanh',
+    'Cbrt': 'Math.cbrt',
     'ceiling': 'Math.ceil',
     'cos': 'Math.cos',
     'cosh': 'Math.cosh',
     'exp': 'Math.exp',
+    'expm1': 'Math.expm1',
     'floor': 'Math.floor',
     'log': 'Math.log',
     'Max': 'Math.max',
     'Min': 'Math.min',
     'sign': 'Math.sign',
     'sin': 'Math.sin',
     'sinh': 'Math.sinh',
+    'Sqrt': 'Math.sqrt',
     'tan': 'Math.tan',
     'tanh': 'Math.tanh',
 }
 
 
-class JavascriptCodePrinter(CodePrinter):
+class AbstractJavaFamilyPrinter(CodePrinter):
+
+    def _print_Pow(self, expr):
```

Comment:
Is this the only thing they have in common or is there more coming? It doesn't seem worth having a shard superclass for just one method.